src/commands/app/clean-build.js

```diff
@@ -0,0 +1,110 @@
+/*
+Copyright 2025 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const ora = require('ora')
+const chalk = require('chalk')
+const fs = require('fs-extra')
+const path = require('path')
+
+const BaseCommand = require('../../BaseCommand')
+const aioLogger = require('@adobe/aio-lib-core-logging')('@adobe/aio-cli-plugin-app:clean-build', { provider: 'debug' })
+
+class CleanBuild extends BaseCommand {
+  async run () {
+    const { flags } = await this.parse(CleanBuild)
+    const configs = await this.getAppExtConfigs(flags)
+
+    const spinner = ora()
+
+    try {
+      for (const [name, config] of Object.entries(configs)) {
+        await this.cleanAllBuildArtifacts(name, config, spinner)
+      }
+
+      this.log(chalk.green(chalk.bold('Build artifacts cleaned up successfully!')))
+    } catch (error) {
+      spinner.stop()
+      throw error
+    }
+  }
+
+  async cleanAllBuildArtifacts (name, config, spinner) {
+    try {
+      spinner.start(`Cleaning build artifacts for '${name}'`)
+
+      if (!config || !config.root) {
+        spinner.info(`No root directory found for extension '${name}'`)
+        return
+      }
+
+      const locations = []
+
+      if (config.app && config.app.dist && fs.existsSync(config.app.dist)) {
+        locations.push(config.app.dist)
+      }
+
+      if (config.dist && fs.existsSync(config.dist)) {
+        locations.push(config.dist)
+      }
+
+      const rootDist = path.join(config.root, 'dist')
+      if (fs.existsSync(rootDist)) {
+        locations.push(rootDist)
+      }
+
+      aioLogger.debug(`Cleaning locations for ${name}: ${locations.join(', ')}`)
+
+      // Clean all possible locations
+      for (const location of locations) {
+        // First try to remove tracking files explicitly
+        const builtPath = path.join(location, 'last-built-actions.json')
+        const deployedPath = path.join(location, 'last-deployed-actions.json')
```

Comment:
can we put these two tokens as constants somewhere? And also update the other locations to use these constants. I'm not sure if they are in the app plugin.

Reply:
Done